platform/kube/inject/testdata/kube-system.yaml.injected

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  initializers:
+    pending:
+    - name: sidecar.initializer.istio.io
+  name: hello
+  namespace: kube-system
+spec:
+  replicas: 7
+  strategy: {}
```

Comment:
Why include some empty values here? Can we remove this? Ditto for others.

I also noticed that some old files also include such kind of empty values, is it really useful?